packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -36,13 +36,18 @@ export default class HostingGeneralInformationsCtrl {
     this.HostingRuntimes = HostingRuntimes;
     this.hostingSSLCertificate = hostingSSLCertificate;
     this.OvhApiScreenshot = OvhApiScreenshot;
+
+    this.goToDetachEmail = $scope.$parent.$ctrl.goToDetachEmail;
```

Comment:
Why `$scope.$parent` ? Could this just be in parent `$scope` and forwarded to children ?